test/fit_tests/deploy/os_ova_install.py

```diff
@@ -4,7 +4,11 @@
 Author(s):
 George Paulos
 
-This script installs blank OVA template.
+This script installs blank OVA templates into selected stack.
+if numvms is greater than 1, MAC address installation on OVA is done by stack number convention
```

Comment:
We should describe here what the "stack number convention" is:
OUI:00:stack-number:VM-number